IAM/Frequently_asked_questions.mediawiki

```diff
@@ -0,0 +1,49 @@
+=== Mozilla IAM FAQ (Frequently Asked Questions) ===
+
+==== '''Q''': ''What is Mozilla IAM?'' ====
+
+Mozilla IAM stands for Mozilla's Identity and Access Management. It's the internal login system that Mozilla uses to login to various web properties and systems.
+
+Usually, you'd use Mozilla IAM as a Mozilla Employee, or has a contributor with access to the tools and resources Mozilla uses day to day.
+An example of that would be Air Mozilla: https://air.mozilla.org
+
+==== '''Q''': ''How do I login with Mozilla IAM?'' ====
+
+Mozilla IAM supports various login methods, such as "LDAP" (Employee logins), GitHub social login, Google social login and email login (which we call "passwordless").
+Certain methods support and enforce the use of a 2nd factor for authentication and may grant access to more sensitive services.
+
+==== '''Q''': ''Why is my login failing with an error message telling me to use "GitHub/Google/LDAP/etc" instead?'' ====
+
+If your login (your primary email address used by Mozilla IAM) matches an existing account which provides higher security (for example, which enforce the use of 2nd factors for authentication), we require that you use the most secure method available to login.
+
+==== '''Q''': ''Why do you support email login ("passwordless") if it's less safe than other methods?'' ====
+
+Sometimes all you want to do is post a comment on a public forum. For that, we often need to provide a valid identity, but we also want to make it as easy as possible for you to contribute.
```

Comment:
Is this a good answer? I might have said something like: Passwordless email login via Auth0 was the most appropriate alternative to the decommissioning of Persona at the end of 2016. It allows us to authenticate volunteers in a cost efficient way.

The above statement is very raw. I would like to know if this is a direction you would want to take at all?
The reason for changing it is because the current answer makes it sound as if you would be able to continue using passwordless email login for Discourse posts. This is not correct for Staff members, because we always force the "most secure method". Right?

Reply:
if you have no context, you have no idea what Persona is/was

Reply:
as staff you can create a passwordless identity where you login for public discussions instead of LDAP. it just cannot match the LDAP login